terraform/app/modules/cloudfront/variables.tf

```diff
@@ -47,4 +47,10 @@ locals {
   cloudfront_aliases_cnames                              = [for zone in var.route53_zones : "${var.route53_cname_record}.${zone}"]
   cloudfront_aliases                                     = concat(var.route53_a_records, local.cloudfront_aliases_cnames)
   cloudfront_viewer_certificate_minimum_protocol_version = "TLSv1.2_2018"
+  cloudfront_path_pattern                                = { pattern_1 = { path = "/packs/*" }, pattern_2 = { path = "/attachment*" } }
```

Comment:
Since there is only 1 value for each iteration, we could use a simple list for cloudfront_path_pattern:
```
cloudfront_path_pattern = ["/packs/*", "/attachment*"]
```

Reply:
Also these are specifically for cached content, so a more explicit variable name may be used such as `cloudfront_cached_paths`